Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -132,8 +132,8 @@ - (instancetype)initWithArticleTitle:(MWKTitle*)title
 
 - (void)showLanguagePicker {
     LanguagesViewController* languagesVC = [LanguagesViewController wmf_initialViewControllerFromClassStoryboard];
-    languagesVC.downloadLanguagesForCurrentArticle = YES;
-    languagesVC.languageSelectionDelegate          = self;
+    languagesVC.articleTitle              = self.articleTitle;
```

Comment:
you could be a good boyscout by drying up essentially this whole method into a `LanguagesViewController` class method